### PR TITLE
PLA-1528: add possibility to add release tag

### PR DESCRIPTION
**Pull request by majdhasan**
## Check yourself

- [ ] Code is linted
- [ ] Tested
- [ ] All new/updated code is covered with tests

## Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### CODE REVIEW

File: .github/workflows/documentation.yaml [hunk 1]
```diff
@@ -27,6 +30,6 @@ jobs:
       - name: Documentation push
         run: |
           cd ${{ inputs.OPENAPI_DEFINITION_PATH || './openapi' }}
-          redocly bundle ${{ inputs.PROJECT_TAG }}@${{ inputs.RELEASE_TAG_VERSION }} --output dist/${{ inputs.RELEASE_TAG_VERSION }}.yaml
+          redocly bundle ${{ inputs.PROJECT_TAG }}@${{ inputs.DESTINATION_TAG || inputs.RELEASE_TAG_VERSION }} --output dist/${{ inputs.RELEASE_TAG_VERSION }}.yaml
```
Comment (dasmarcomoreira) [inline .github/workflows/documentation.yaml:33]: make it as such


"redocly bundle ${{ inputs.PROJECT_TAG }}@${{ inputs.SOURCE_TAG }} --output dist/${{ inputs.DESTINATION_TAG }}.yaml"

File: .github/workflows/documentation.yaml [hunk 2]
```diff
@@ -9,6 +9,9 @@ on:
       PROJECT_TAG:
         required: true
         type: string
+      DESTINATION_TAG:
```
Comment (dasmarcomoreira) [inline .github/workflows/documentation.yaml:12]: DELETE THIS

Comment (dasmarcomoreira) [inline .github/workflows/documentation.yaml:12]: Rename to SOURCE_TAG

**Review by dasmarcomoreira — CHANGES_REQUESTED** [review]
RC

File: .github/workflows/documentation.yaml (hunk 2 above)
Comment (majdhasan) [reply .github/workflows/documentation.yaml:12]: I would rather avoid breaking changes, wdyt

File: .github/workflows/documentation.yaml (hunk 1 above)
Comment (majdhasan) [reply .github/workflows/documentation.yaml:33]: I think this is better --output dist/${{ inputs.**SOURCE_TAG** }}.yaml"

Comment (majdhasan) [reply .github/workflows/documentation.yaml:33]: so:
redocly bundle ${{ inputs.PROJECT_TAG }}@${{ inputs.SOURCE_TAG }} --output dist/${{ inputs.SOURCE_TAG }}.yaml

File: .github/workflows/documentation.yaml [hunk 3]
```diff
@@ -3,12 +3,15 @@ name: Stage Documentation
...
-      RELEASE_TAG_VERSION:
+      PROJECT_TAG:
         required: true
         type: string
-      PROJECT_TAG:
+      SOURCE_TAG:
         required: true
         type: string
+      DESTINATION_TAG:
+        required: false
```
Comment (dasmarcomoreira) [inline .github/workflows/documentation.yaml:13]: make it required

**Review by dasmarcomoreira — APPROVED** [review]
Dont forget to create a new tag (v1.7.0 I beilieve)

**Pull request merged**